ios/RNTrackPlayer/RNTrackPlayer.swift

```diff
@@ -149,15 +149,19 @@ public class RNTrackPlayer: RCTEventEmitter {
         let autoWait: Bool = config["waitForBuffer"] as? Bool ?? false
         player.automaticallyWaitsToMinimizeStalling = autoWait
         
+        // configure if control center metdata should auto update
+        let autoUpdateMetadata: Bool = config["automaticallyUpdateNowPlayingInfo"] as? Bool ?? true
```

Comment:
This parameter looks too long, how about something like `autoMetadataUpdate`?

Reply:
Changed to `autoUpdateMetadata`